src/components/CreateMatch/Team/Team.js

```diff
@@ -1,17 +1,42 @@
-import React from 'react';
-import CreateTeam from './CreateTeam/CreateTeam';
+import React, { Component } from 'react';
 import search from '../../../images/search-icon.png';
+import addTeamList from '../../../images/create-team-list.png';
+import check from '../../../images/check-icon.png';
+import newPlayer from '../../../images/add-player.png';
+import ballPlay from '../../../images/ball-play.png';
+import captain from '../../../images/captain.png';
 import './Team.scss';
 
-const Team = () => {
-    return(
-        <div>
-            <CreateTeam />
-            <span>
-                <input placeholder="Mi Equipo"></input>
-                <img scr={search} alt="buscar Mi Equipo"/>
-            </span>
-        </div>
-    )
+
+class Team extends Component {
+
+    render() {
+        return(
+            <div className="team">
+                {this.props.createTeam ? <span className="info">
```

Comment:
Use object destructuring for this.props variables for example createTeam and showCreateTeam props.